auth/auth.go

```diff
@@ -10,8 +10,10 @@ import (
 	"gorm.io/gorm/clause"
 
 	contractsauth "github.com/goravel/framework/contracts/auth"
+	"github.com/goravel/framework/contracts/cache"
+	"github.com/goravel/framework/contracts/config"
+	"github.com/goravel/framework/contracts/database/orm"
 	"github.com/goravel/framework/contracts/http"
-	"github.com/goravel/framework/facades"
```

Comment:
All of the modules shouldn't be dependent on facades and should inject if needed.